index.html

```diff
@@ -4,173 +4,155 @@
-    <section class="projects">
-        
-        <div class="gallery">
-<h2 class="project-title card1">Projects</h2>
-
-            <div class="caja1-1" card></div> 
-            <article class="card card2">
-                <img class="imgp" src="images/project1.png" alt="project1" width="320px">
-                
-                <div class="pData">
-                    <h3>Project name goes here</h3>
-                    <ul class="projectUl">
-                        <li class="projectLi">HTML/CSS</li>
-                        <li class="projectLi">Ruby on Rails</li>
-                        <li class="projectLi">Java script</li>
-                    </ul>
-                    <button class="see">
-                        See this project &nbsp →
-                    </button>
-                </div>
-
-
-
-
-
-            </article>
-            
-            <article class="card card3">
-                <img src="images/project2.png" alt="project2" width="100%">
-
-
-
-
-                
-            </article>
-            <div class="card card4"></div>
-            <article class="card cardx">
-                <img src="images/project6.png" alt="project3" width="100%">
-                <div class="pData">
-                    <h3>Project name goes here</h3>
-                    <ul class="projectUl">
-                        <li class="projectLi">HTML/CSS</li>
-                        <li class="projectLi">Ruby on Rails</li>
-                        <li class="projectLi">Java script</li>
-                    </ul>
-                    <button class="see">
-                        See this project &nbsp →
-                    </button>
-                </div>
-            </article>
-            
-            <article class="card card5">
-                <img src="images/project4.png" alt="project4" width="100%">
-                
-            </article>
-            <article class="card card6">
-                <img src="images/project3.png" alt="project5" width="100%">
-                
-            </article>
-            <div class="card card7"></div>
-            <div class="card card8"></div>
-            <article class="card card9">
-                <img src="images/project5.png" alt="project6" width="100%">
-                
-            </article>
-            </div>
-    </section>
+    </nav>
+   </header>
+
+
+   <div class="intro-container">
+    <div class="intro">
+        <h2 class="intro-greeting">Welcome to FEMUA 15</h2>
+        <h1 class="intro-title"> the Anoumabo Urban Music Festival (FEMUA) <span class="year">2023<span></h1>
+        <h3 class="intro-text"> FEMUA, which defines itself as one of the biggest festivals in sub-Saharan Africa, is held annually in a cosmopolitan city.</h3>
+        <h2 class="intro-date">2023.11.27(MON) ~ 29(SAT)</h2>
+        <h3 class="intro-place">@ BOUAKE, Peace Stadium</h3>
+    </div>
+</div>
+<div class="background">
+</div>
+<div class="background-gradient">
+</div>
```

Comment:
Kindly adjust the hero section to be as close to the design as possible. Note the following points:

- The fonts seem to be differ
- The section has a background image and a semi-transparent white layer

Your design:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/842bdf7e-7763-4b97-83d7-e8f2a2818ae4)

Figma design:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/0337a592-0ae8-4ce7-9caf-ea7f42da08c4)